Add Dev Container section on README (#694)
dermotduffy/frigate-hass-card: README.md

Resultant build will be at `dist/frigate-hass-card.js`. This could be installed via the [manual installation instructions above](#manual-installation).

### Dev Container

> [![Open in Remote - Containers](https://img.shields.io/static/v1?label=Remote%20-%20Containers&message=Open&color=blue&logo=visualstudiocode)](https://vscode.dev/redirect?url=vscode://ms-vscode-remote.remote-containers/cloneInVolume?url=https://github.com/dermotduffy/frigate-hass-card)

Reviewer: To match the other badges used in the README:

```suggestion
[![Open in Remote - Containers](https://img.shields.io/static/v1?label=Remote%20-%20Containers&message=Open&color=blue&logo=visualstudiocode&style=flat-square)](https://vscode.dev/redirect?url=vscode://ms-vscode-remote.remote-containers/cloneInVolume?url=https://github.com/dermotduffy/frigate-hass-card)
```

You can use the [VS Code Remote - Containers](https://code.visualstudio.com/docs/remote/containers) extension to speed up the development environment creation. Simply:

1. Clone the repository to your machine
1. Open VS Code on it
1. Reopen the folder in the Dev Container
1. Once done, press `F5` to start debugging

> Ideally, everything should be working without any additional configuration.
> 
> Under the hoods, the dev container setup takes care of bringing up:

Reviewer: ```suggestion
Everything should just work without any additional configuration. Under the hood, the dev container setup takes care of bringing up:
```

Resultant build will be at `dist/frigate-hass-card.js`. This could be installed via the [manual installation instructions above](#manual-installation).

### Dev Container

[![Open in Remote - Containers](https://img.shields.io/static/v1?label=Remote%20-%20Containers&message=Open&color=blue&logo=visualstudiocode)](https://vscode.dev/redirect?url=vscode://ms-vscode-remote.remote-containers/cloneInVolume?url=https://github.com/dermotduffy/frigate-hass-card)

You can use the [VS Code Remote - Containers](https://code.visualstudio.com/docs/remote/containers) extension to speed up the development environment creation. Simply:

> 1. Clone the repository in your machine

Reviewer: ```suggestion
1. Clone the repository to your machine
```

2. Open VS Code on it
3. Reopen the folder in the Dev Container
4. Once done, press `F5` to start debugging

>    >**NOTE**: Use `admin`/`admin` to log into Home Assistant.

Reviewer: ```suggestion
```

(Not necessary after my other PR)

Ideally, everything should be working without any additional configuration.

Under the hoods, the dev container setup takes care of bringing up:

* Home Assistant (port `48123:8123`)
* Frigate (ports `45000:5000`, `41935:1935`)
* MQTT (port `41883:1883`)

As docker-compose containers.

> * The Frigate Home Assistant Integration is registerd as a `git submodule` at `.devcontainer/frigate-hass-integration`, and VS Code will initialize/clone as well for you.

Reviewer: ```suggestion
* The Frigate Home Assistant Integration is registered as a `git submodule` at `.devcontainer/frigate-hass-integration`, and VS Code will initialize/clone it for you.
```
